docs/content/development/extensions-core/kafka-extraction-namespace.md

```diff
@@ -25,7 +25,7 @@ If you need updates to populate as promptly as possible, it is possible to plug
 |`kafkaTopic`|The kafka topic to read the data from|Yes||
 |`kafkaProperties`|Kafka consumer properties. At least"zookeeper.connect" must be specified. Only the zookeeper connector is supported|Yes||
 |`connectTimeout`|How long to wait for an initial connection|No|`0` (do not wait)|
-|`isOneToOne`|The map is a one-to-one (see[Lookup DimensionSpecs](../querying/dimensionspecs.html))|No|`false`|
+|`isOneToOne`|The map is a one-to-one (see[Lookup DimensionSpecs](../../querying/dimensionspecs.html))|No|`false`|
```

Comment:
Would you please add a space between `see` and `Lookup DimensionSpecs`?

Reply:
done.